ansible/roles/ooniprobe/tasks/install-git.yml

```diff
@@ -34,4 +13,4 @@
     name={{ item }}
   with_items:
     - cffi
-    - ooniprobe
+    - 'git+https://github.com/TheTorProject/ooni-probe.git#egg=ooniprobe'
```

Comment:
Let's be careful with this. Since it leads to the installation of a potentially unstable version of ooniprobe. I would make it a switch to install the master branch version vs installing it from the pypi release.